src/testController.ts

```diff
@@ -106,8 +110,14 @@ export class TestController {
 
       // Add test methods as children to the test class so it appears nested in Test explorer
       // and running the test class will run all of the test methods
-      if (name.startsWith("test_")) {
-        testItem.tags = [this.debugTag];
+
+      // Older Ruby LSP versions may not include 'kind' so we try infer it from the name.
+      if (!res.data?.kind && name.startsWith("test_")) {
+        res.data.kind = "example";
+      }
+
+      if (res.data?.kind === "example") {
```

Comment:
If we always assign `kind` to tags in the above conditions, we can simplify this to just check the first tag instead of `res.data`?

Reply:
I've updated it to check the tags instead of `res.data`, but relying on its position in the array seems fragile so I've used a `find`.